app.js

```diff
@@ -3,12 +3,9 @@ const express = require('express');
 const path = require('path');
 const logger = require('morgan');
 const app = express();
+const indexRoute = require('./routes/index');
+const pizzaRoute = require('./routes/pizza')
 
-/* setting up port & listen */
-const PORT = process.env.PORT || 3000;
-app.listen(PORT, function() {
-  console.log(`listening on port ${PORT}`);
-});
 
 /* set the view engine */
 app.set('views', './views');
```

Comment:
No. I know this isn't your code but this comment is wrong. This line is not doing anything. (It sets where it looks for the views directory to `views`, which incidentally it already is by default). In order to actually set the view engine, do this:
```
app.set("view engine", "ejs");
```